www/js/controllers/region-home-controller.js

```diff
@@ -689,6 +690,10 @@ angular.module('starter.controllers')
       return;
     }
 
+    if($scope.inputUser.email!=null && $scope.inputUser.email!="" && ($scope.inputUser.email.indexOf(".")==-1 || $scope.inputUser.email.indexOf("@")==-1 || $scope.inputUser.email.indexOf("@")<$scope.inputUser.email.indexOf("."))) {
```

Comment:
Move this to utility where you can call a method to find out whether email is valid.